File: backend/src/main/java/com/imagosur/terminal_autoconsulta/model/response/Link.java
package com.imagosur.terminal_autoconsulta.model.response;

public class Link {

	private String url;
	private Boolean connect = false;
	
	public Link(String option) {
		if(option.equals("correo"))
			this.connect = true;
		this.url = "";
	}
	
	public Link() {
		this.url = "";
	}

	public Boolean getConnect() {
		return connect || !this.url.isEmpty();
	}

	public void setConnect(Boolean connect) {
		this.connect = connect;
	}

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}
	
	
	
}

File: frontend/src/renderer/modelos/Link.js
class Link{

    url;
    static instancia;

    static instanciar(){
        if(!this.instancia)
            this.instancia = new Link();
        return this.instancia;
    }

    getUrl(){
        return this.url;
    }

    setUrl(url){
        this.url = url;
    }

}
export default Link;